src/core/Curl/Handler.php

```diff
@@ -412,6 +422,20 @@ private function setOption(int $opt, $value): bool
                 $this->nobody = boolval($value);
                 $this->method = 'HEAD';
                 break;
+            case CURLOPT_RESOLVE:
+                foreach ((array) $value as $resolve) {
+                    if (substr($resolve, 0, 1) === '+') {
```

Comment:
Better add another check here:

```php
                    if (substr($resolve, 0, 1) === '-') {
                        continue;
                    }
```

The official doc says that ([Here](https://curl.se/libcurl/c/CURLOPT_RESOLVE.html)):

> Remove names from the DNS cache again, to stop providing these fake resolves, by including a string in the linked list that uses the format "-HOST:PORT". The host name must be prefixed with a dash, and the host name and port number must exactly match what was already added previously.

Reply:
done.